### PR TITLE
refactor query to just count all  including Empty/Dummy Operators

### PR DESCRIPTION
**Pull request by bote795**
we are only excluding: astronomer_monitoring_dag now

### CODE REVIEW

**Comment by bote795** [comment]
after discussing the following choices:
So you either:
- do an inner join and miss out on any runs where the TI was deleted
- do a left join and you can’t exclude deleted TI’s that used Empty/Dummy
- count DummyOperator/EmptyOperator completely 

we decided to go with count Dummy operator/Empty operator cc: @cmarteepants

File: analytics_plugin.py [hunk 1]
```diff
@@ -50,23 +50,17 @@ def tasks_report_query(session, start_date, end_date) -> Any:
...
-                TaskInstance.task_id == Log.task_id,
+        .filter(
+            Log.dttm >= start_date,
+            Log.dttm <= end_date,
+            Log.dag_id != "astronomer_monitoring_dag",
+            Log.event.in_(
+                [
+                    TaskInstanceState.SUCCESS,
+                    TaskInstanceState.FAILED,
+                ]
```
Comment (jedcunningham) [inline analytics_plugin.py:61]:
```suggestion
                ],
            Log.execution_date is not None,
```

or

```suggestion
                ],
            Log.extra is None,
```

Just experimenting a bit and noticed that marking TIs lead to audit log entries too. These are the obvious differences. Owner also varies, but (I suspect) if the TI owner and UI user name matches, you wouldn't be able to use that to differentiate.

**Review by jedcunningham — COMMENTED** [review]
No test coverage for this query?

**Comment by bote795** [comment]
> No test coverage for this query?

got distracted when i was going to add the tests going to do that now

update we mock out the query so no tests are needed unless we did E2E not sure what we would test

File: analytics_plugin.py (hunk 1 above)
Comment (bote795) [reply analytics_plugin.py:61]: so exeuction_date not being set says it from the UI what does the extra is none do?

Comment (jedcunningham) [reply analytics_plugin.py:61]: I noticed 3 differences between the mark and task run. I don't think owner is good enough to check (plus it'd be difficult to do).

Screen Shot 2022-12-02 at 9 25 16 AM

Comment (bote795) [reply analytics_plugin.py:61]: ahh thx for that clarification! ill add those in

Comment (bote795) [reply analytics_plugin.py:61]: @cmarteepants thoughts?

Comment (cmarteepants) [reply analytics_plugin.py:61]: Makes sense to me. Guessing that the extra field contains TI attributes that were used when marking the run with success.

Comment (bote795) [reply analytics_plugin.py:61]: so we do want to remove UI mark ones? @cmarteepants

Comment (cmarteepants) [reply analytics_plugin.py:61]: Yes please

**Comment by rishkarajgi** [comment]
Link to issue - https://github.com/astronomer/issues/issues/5272

**Pull request merged**